tests/dat/actions/CacheConcurrencyTests.sh

```diff
@@ -0,0 +1,94 @@
+(for i in `seq 1 $N`; do ($WSK action create testy$i "${ACTIONFILE}" &); done; wait) 2>&1 | grep -v failure > /dev/null
+
+if [ $? == 1 ]; then
+    echo "FAIL"
+    exit 1
+else
+    echo "PASS"
+fi
+
+echo -n "update "
+(for i in `seq 1 $N`; do ($WSK action update testy$i -p p v &); done; wait) 2>&1 | grep -v error > /dev/null
+
+if [ $? == 1 ]; then
+    echo "FAIL in update"
+    exit 1
+else
+    echo "PASS"
+fi
+
+echo -n "delete+get "
+(for i in `seq 1 $N`; do ($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action delete testy$i &); ($WSK action get testy$i &); ($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &); done; wait) 2>&1 | grep -v "unable to delete" > /dev/null
+
+if [ $? == 1 ]; then
+    echo "FAIL"
+    exit 1
+else
+    echo "PASS"
+fi
+
+echo -n "get after delete "
+(for i in `seq 1 $N`; do ($WSK action get testy$i &); done; wait) 2>&1 | grep ok > /dev/null
+
+if [ $? == 0 ]; then
+    echo "FAIL"
+    exit 1
+else
+    echo "PASS"
+fi
+
+
+echo -n "create "
+(for i in `seq 1 $N`; do ($WSK action create testy$i "${ACTIONFILE}" &); done; wait) 2>&1 | grep -v failure > /dev/null
+
+if [ $? == 1 ]; then
+    echo "FAIL"
+    exit 1
+else
+    echo "PASS"
+fi
+
+for i in `seq 1 $N`; do $WSK action update testy$i -p smurf zoomba >& /dev/null; done
+
+echo -n "update+get "
+(for i in `seq 1 $N`; do ($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action update testy$i -p smurf blue &); ($WSK action get testy$i &); ($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &);($WSK action get testy$i &); done; wait) 2>&1 | grep "ok: updated action" > /dev/null
+
+if [ $? == 1 ]; then
+    echo "FAIL"
+    exit 1
+else
+    echo "PASS"
+fi
+
+echo -n "get after update+get "
+for i in `seq 1 $N`; do
+    $WSK action get testy$i | grep zoomba > /dev/null
+    if [ $? == 0 ]; then
+	echo "FAIL"
+	exit 1
+    fi
+done
+echo "PASS"
+
```

Comment:
partly because i wanted to be able to debug the test against YP and localhost (verifying failure on YP, and success on localhost), and partly because doing so required about 12 debug iterations to find the spot that'd give me a valid test -- i didn't have the patience to wait 12*30 minutes that it would take to do this with `tests:test`